appveyor/requirements.txt

```diff
@@ -4,3 +4,4 @@
 numpy==1.9.2
 nose
```

Comment:
It's so frustrating: nose is only needed for the with_setup. We need to find a way around this.